openghg/retrieve/icos/_retrieve.py

```diff
@@ -49,6 +50,8 @@ def retrieve_atmospheric(
                 - "never" - don't update mismatches and raise an AttrMismatchError
                 - "from_source" / "attributes" - update mismatches based on attributes from ICOS Header
                 - "from_definition" / "metadata" - update mismatches based on input metadata
+        force: If True, disregard previously retrieved hashes. Use this if you are unable to re-add
```

Comment:
In this circumstance this comment is true but the key can also be used to replace identical data as well.
On the `new_object_store` branch this key word is defined as
> force: Force adding of data even if this is identical to data stored.

and I think that does fit with this definition as well so could combine the two in some way like this:

```suggestion
        force: Force adding of data even if this is identical to data stored. This can also be used to disregard previously retrieved hashes if unable to re-add data.
```